### PR TITLE
Latest updates to the patient home page for a sub-study subject

### PR DESCRIPTION
**Pull request by achen2401**
Addresses: 
https://jira.movember.com/browse/TN-2786
https://jira.movember.com/browse/TN-2644
update patient home page for a sub-study subject based on latest content and design changes

Changes include:

- Verbiage updates to sub-study tiles
- update layout for when both main study and sub-study questionnaires are completed
- update sub-study thank you modal based on latest design and content changes
- fix tile text for when **both** sub-study and main-study tiles are present
- put back text for the main study questionnaire completed tile
- remove un-needed extra HTML tags from assessment_engine.html

### CODE REVIEW

File: portal/eproms/templates/eproms/assessment_engine/ae_macros.html [hunk 1]
```diff
@@ -101,96 +101,121 @@ <h4 class="portal-description-title">
...
+            title_text = _("%(substudy_title)s: View how you are doing over time", substudy_title=substudy_title),
+            card_class = card_class if card_class else "portal-description-complete") -%}
             <p>{{_("View your longitudinal charts and discuss with your care provider")}}</p>
         {%- endcall -%}
         {{empro_thankyou_modal(user)}}
     {%- endif -%}
 {%- endmacro -%}
+{%- macro empro_thankyou_card(full_name="") -%}
+    {% call render_header(title_text=_("Hi, %(full_name)s.", full_name=full_name if full_name else '')) %}
+        <p>{{_("None of your questionnaires are due.  You can view your completed questionnaires below:")}}</p>
```
Comment (ivan-c) [inline portal/eproms/templates/eproms/assessment_engine/ae_macros.html:130]: Extra space?
```suggestion
        <p>{{_("None of your questionnaires are due. You can view your completed questionnaires below:")}}</p>
```

File: portal/eproms/templates/eproms/assessment_engine/ae_macros.html [hunk 2]
```diff
@@ -101,96 +101,121 @@ <h4 class="portal-description-title">
...
+        <h4>{{_("Your Support Team")}}</h4>
+        <p>{{_("You reported having some particular challenges with: ")}}</p>
+        <ul class="triggersDisplayList">
+           <!-- dynamically populated based on user trigger domain(s) -->
+        </ul>
+        <br/>
+        <p><b>{{_("To help address these, we've informed your care team and they'll be in contact with you soon.")}}</b></p>
+        <br/>
+        {% if user and user.organizations %}
+            <p>{{_("In the meantime, if you have any questions or need assistance, please contact your team at %(organization) directly.  They're happy to help.", organization=organization)}}</p>
```
Comment (ivan-c) [inline portal/eproms/templates/eproms/assessment_engine/ae_macros.html:151]:
```suggestion
            <p>{{_("In the meantime, if you have any questions or need assistance, please contact your team at %(organization) directly. They're happy to help.", organization=organization)}}</p>
```

File: portal/eproms/templates/eproms/assessment_engine/ae_macros.html [hunk 3]
```diff
@@ -101,96 +101,121 @@ <h4 class="portal-description-title">
...
+        {% endif %}
+    </div>
+{%- endmacro -%}
+{%- macro empro_modal_default_report_block() -%}
+    <div class="item">
+        <h4>{{_("Your Report")}}</h4>
+        <p>{{_("Here’s where you'll find an ongoing summary of your responses, each time you complete a questionnaire.")}}</p>
```
Comment (ivan-c) [inline portal/eproms/templates/eproms/assessment_engine/ae_macros.html:158]:
```suggestion
        <p>{{_("Here's where you'll find an ongoing summary of your responses, each time you complete a questionnaire.")}}</p>
```

**Review by ivan-c — APPROVED** [review]
Few verbiage nitpicks; ready to merge otherwise!

File: portal/eproms/templates/eproms/assessment_engine/ae_macros.html (hunk 1 above)
Comment (achen2401) [reply portal/eproms/templates/eproms/assessment_engine/ae_macros.html]: thank you, Ivan, fixed

File: portal/eproms/templates/eproms/assessment_engine/ae_macros.html (hunk 2 above)
Comment (achen2401) [reply portal/eproms/templates/eproms/assessment_engine/ae_macros.html]: thank you, fixed.

File: portal/eproms/templates/eproms/assessment_engine/ae_macros.html (hunk 3 above)
Comment (achen2401) [reply portal/eproms/templates/eproms/assessment_engine/ae_macros.html]: thank you, fixed

**Pull request merged**